src/main/resources/io/github/triglav_dataflow/jenkins/trigger/polling_triglav/PollingTriglavTrigger/config.jelly

```diff
@@ -33,6 +33,9 @@
   <f:entry title="Time Unit" field="timeUnit">
     <f:select/>
   </f:entry>
+  <f:entry title="Alternative Execution Time" field="alternativeExecutionTime">
+    <f:textbox value="${it.alternativeExecutionTime}" checkUrl="'descriptorByName/io.github.triglav_dataflow.jenkins.trigger.polling_triglav.PollingTriglavTrigger/checkAlternativeExecutionTime?alternativeExecutionTime='+escape(this.value)"/>
```

Comment:
I think `checkUrl` is not required. ref. https://wiki.jenkins.io/display/JENKINS/Basic+guide+to+Jelly+usage+in+Jenkins#BasicguidetoJellyusageinJenkins-Formvalidation

> You can write doCheckFIELD method on your descriptor to add the form validation logic.

Reply:
Thanks, I checked and its working correctly without it. It was handled by @DataBoundConstructor